README.md

```diff
@@ -1,2 +1,19 @@
 # search-spider
 The home for the spider that supports search.gov
+
+### Scrapy Web Crawler
+The spider can either scrape for URLs from the list of required domains or take in a domain and starting URL to scrape 
+site/domain.
+
+## Running Against All Listed Search.gov Domains
+Navigate down to `search-gov-indexing/spider-foundational-work/search_gov_spiders/search_gov_spiders/spiders/`, then
```

Comment:
@chancecrawford Before I get deep down in the weeds in reviewing this, my initial big picture questions (partially triggered by this sentence) is - now that we've got a separate, single purpose repo for this work - can we flatten out some of this wacky directory nesting now from the start?

Reply:
Done 🙃